com.github.FontManager.FontManager.yaml

```diff
@@ -0,0 +1,91 @@
+app-id: com.github.FontManager.FontManager
+runtime: org.gnome.Platform
+runtime-version: '46'
+sdk: org.gnome.Sdk
+
+command: font-manager
+
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --share=network
+
+  # XDG Data/Fonts location
+  - --filesystem=xdg-data/fonts
```

Comment:
Needs exception

Reply:
Do I have to submit the exception, or is that something out of my direct control?

Reply:
You have to submit it https://docs.flathub.org/docs/for-app-authors/linter#exceptions but the id thing should be figured out first. 

You can run the linter on the manifest to get the error code. See #3 https://docs.flathub.org/docs/for-app-authors/submission#before-submission